content/_data/changelogs/weekly.yml

```diff
@@ -268,11 +268,11 @@
   date: 2017-04-02
   changes:
     - type: major bug
-      issue: 42744
-      pull: 2826
       message: >
-        Update to Windows Service Wrapper <tt>2.0.3</tt> and Windows Agent Installer <tt>1.8</tt> to prevent conversion of environment variables to lowercase in the agent executable, regression in Jenkins <tt>2.50</tt>.
+        Update to Windows Service Wrapper 2.0.3 and Windows Agent Installer 1.8 to prevent conversion of environment variables to lowercase in the agent executable, regression in Jenkins 2.50.
       references:
+        - issue: 42744
+        # pull: 2826
```

Comment:
Deliberate: On top level, we only show issue if both exist, but in `references`, we show everything. Keep top-level behavior but document the pull request number.